### DISCUSSION TITLE
Automation between light mode and dark mode

### DISCUSSION THREAD

**Discussion by Yumega** [darkreader/darkreader#8473] [Q&A]
It's convenient to turn on/off Darkreader automatically through setting option,
but I prefer light mode in daytime, dark mode at night, not turn off the Dark reader.
Is it possible change light mode and dark mode automatically?

**Comment by Gusted** [comment]
See #1713

**Reply by Yumega** [reply]
> See #1713

sorry, i dont understand at all , it looks super complicated to meet my need

**Reply by Gusted** [reply]
Okay, follow the following steps:


    Click Dark Reader icon.
    Click Dev tools (in the bottom-right corner).
    Click Preview new design.

    Go back to to the popup.
    Click Settings
    Click Automation

**Reply by Yumega** [reply]
> 

Thanks a lot,  really helpful, appreciate your help
